api/src/pcapi/settings.py

```diff
@@ -71,6 +71,7 @@
 CORS_ALLOWED_ORIGINS_BACKOFFICE = os.environ["CORS_ALLOWED_ORIGINS_BACKOFFICE"].split(",")
 CORS_ALLOWED_ORIGINS_NATIVE = os.environ["CORS_ALLOWED_ORIGINS_NATIVE"].split(",")
 CORS_ALLOWED_ORIGINS_ADAGE_IFRAME = os.environ["CORS_ALLOWED_ORIGINS_ADAGE_IFRAME"].split(",")
+SESSION_COOKIE_SAMESITE = str(os.environ.get("SESSION_COOKIE_SAMESITE", "Lax"))
```

Comment:
(détail) `str()` n'est pas nécessaire, car `os.environ.get()` renvoie une chaîne de caractères ou la valeur par défaut si la variable d'environnement n'existe pas. Et comme ici la valeur par défaut est une chaîne de caractères, la conversion est inutile.

```suggestion
SESSION_COOKIE_SAMESITE = os.environ.get("SESSION_COOKIE_SAMESITE", "Lax")
```

Reply:
J'avoue que j'ai mis le str() sans trop réfléchir^^
J'ai fait la modificaiton.